cli/fmt_errors.rs

```diff
@@ -128,9 +128,11 @@ fn format_frame(frame: &JsStackFrame) -> String {
   result
 }
 
+#[allow(clippy::too_many_arguments)]
```

Comment:
😬 please add a TODO to refactor this function

Reply:
eh, the function is called in a single place, and to resolve this we'd need to add a new struct, dont see too much value for that, imo we can leave as-is